service/matching/user_data_manager.go

```diff
@@ -309,7 +309,7 @@ func (m *userDataManagerImpl) fetchUserData(ctx context.Context) error {
 		// one. But if the remote is broken and returns success immediately, we might end up
 		// spinning. So enforce a minimum wait time that increases as long as we keep getting
 		// very fast replies.
-		if elapsed < minWaitTime {
+		if elapsed < m.config.GetUserDataMinWaitTime {
```

Comment:
can you explain this change? I see what this is doing and I can imagine why you want it, but I'm curious why

Reply:
So the intention is to throttle when remote "returns success immediately". `minWaitTime` is not necessarily a small amount as it double each time. I just want to reduce unnecessary delay.